services/openvpn/service/auth_handler_test.go

```diff
@@ -15,71 +15,74 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package session
+package service
 
 import (
 	"testing"
 
 	"github.com/mysteriumnetwork/go-openvpn/openvpn/management"
-	"github.com/mysteriumnetwork/go-openvpn/openvpn/middlewares/server/auth"
 	"github.com/mysteriumnetwork/node/identity"
 	"github.com/mysteriumnetwork/node/session"
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	sessionExistingString = "fake-id"
+)
+
 var (
-	identityToExtract = identity.FromAddress("deadbeef")
-	validator         = mockValidatorWithSession(identityToExtract, session.Session{
-		ID:         session.ID("Boop!"),
-		ConsumerID: identityToExtract,
-	})
+	identityExisting = identity.FromAddress("deadbeef")
+	sessionExisting  = session.Session{
+		ID:         session.ID(sessionExistingString),
+		ConsumerID: identityExisting,
+	}
 )
 
-type fakeAuthenticatorStub struct {
-	username      string
-	password      string
-	called        bool
-	authenticated bool
-}
+func TestValidateReturnsFalseWhenNoSessionFound(t *testing.T) {
+	validator := mockAuthHandler(identity.Identity{}).validate
```

Comment:
Why are you testing mock AuthHandler?

Reply:
It mocks several values to auth handler.
Renamed `mockAuthHandler` -> `createAuthHandler`